setup.py

```diff
@@ -13,6 +13,6 @@
       install_requires=[
           'schematics==1.1.1',
           'requests>=2.13.0',
-          'six==1.11.0',
+          'six>=1.11.0,<=1.12.0',
```

Comment:
@MichaelaShtilmanMinkin I think you forgot to change it here too.

Reply:
Good catch; thanks.